### ISSUE TITLE
Bug with colors of label

### ISSUE THREAD

**Issue by vaneuss1** [opencv/cvat#5038]
Hello. We found such a bug. When we change the label to on an already drawn polygon, the color of the label does not change. It changes only after the page is reloaded.

**Comment by vaneuss1** [comment]
we have release-2.2.0 installed

**Comment by bsekachev** [comment]
Hi @vaneuss1 

This is duplicate of #4966
I will look at this soon.

Please, look to the original issue to track progress.
